### PR TITLE
Update java template

### PR DESCRIPTION
**Pull request by otaviojava**

### CODE REVIEW

File: templates/jenkins/files/README.md [hunk 1]
```diff
@@ -1,19 +1,37 @@
-# Jenkins template for Platform.sh
+# Jenkins for Platform.sh
 
-This project provides a starter kit for Jenkins projects hosted on [Platform.sh](http://platform.sh).
+This project provides a starter kit for Jenkins projects hosted on Platform.sh.
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+Jenkins is an open source automation server written in Java. Jenkins helps to automate the non-human part of the software development process, with continuous integration and facilitating technical aspects of continuous delivery.
```
Comment (Crell) [inline templates/jenkins/files/README.md]: This second part should be included in the description in the template.yaml, and both lines wrapped in p tags.  (Same for all of the templates.)

File: templates/jenkins/files/README.md [hunk 2]
```diff
@@ -1,19 +1,37 @@
...
 
-## Platform.sh configuration files
+## Services
+
+* Java 8
+
+## Post-install
+
+1. After running a Jenkins instance by Platform.sh, the post-installation setup wizard begins.
```
Comment (Crell) [inline templates/jenkins/files/README.md]: "running a Jenkins instance by Platform.sh" - The grammar here is a bit clunky.  I'm not sure what is intended here.  Maybe "After creating a Jenkins project"?  In the others I've specifically called out the first-run behavior.  Is that the case here?

Comment (otaviojava) [reply templates/jenkins/files/README.md]: After creating a Jenkins project is fine.

File: templates/jenkins/files/README.md [hunk 3]
```diff
@@ -1,19 +1,37 @@
...
+2. When you first access a new Jenkins instance, you are asked to unlock it using an automatically-generated password.
+3. Browse to the Jenkins service link and wait until the Unlock Jenkins page appears.
+4. From the Jenkins console log output, copy the automatically-generated alphanumeric password 
+5. Access the Jenkins instance with the (ssh command)[https://docs.platform.sh/development/access-site.html#accessing-the-application-with-ssh]
```
Comment (Crell) [inline templates/jenkins/files/README.md]: Er.  This says there's not one but two passwords to go through?  That's not what I saw when I was testing the template.  I just saw the "go read this file from disk" message.

We can probably collapse several of these steps into a single command: `platform ssh cat /app/.jenkins/secrets/initialAdminPassword`.  (With a link to the CLI docs page where appropriate.)

File: templates/jenkins/files/README.md [hunk 4]
```diff
@@ -1,19 +1,37 @@
...
 * [`.platform/services.yaml`](.platform/services.yaml):  Platform.sh allows you to completely define and configure the topology and [services you want to use on your project](https://docs.platform.sh/configuration/services.html).
 * [`.platform.app.yaml`](.platform.app.yaml): You control your application and the way it will be built and deployed on Platform.sh [via a single configuration file](https://docs.platform.sh/configuration/app-containers.html).
 
 ## References
 
-
 * [Maven](https://maven.apache.org/)
 * [Jenkins](https://jenkins.io/) 
 * [Platform Documentation](https://docs.platform.sh/)
-* [Java at Platform.sh](https://docs.platform.sh/languages/java.html)
```
Comment (Crell) [inline templates/jenkins/files/README.md]: I don't think the link to the docs root is necessary; just the language link and application home page link, in most cases.

File: templates/jenkins/files/README.md (hunk 3 above)
Comment (otaviojava) [reply templates/jenkins/files/README.md]: easier, fixed. Thank you

File: templates/jenkins/files/README.md (hunk 4 above)
Comment (otaviojava) [reply templates/jenkins/files/README.md]: Done, thank you

File: templates/jenkins/files/README.md (hunk 1 above)
Comment (otaviojava) [reply templates/jenkins/files/README.md]: Ok, Done, thank you

File: templates/microprofile-payara/files/README.md [hunk 5]
```diff
@@ -1,14 +1,21 @@
-# Payara Micro Maven template for Platform.sh
+# Payara Micro Maven for Platform.sh
 
-This project provides a starter kit for Payara Micro projects hosted on [Platform.sh](http://platform.sh).
+This project provides a starter kit for Payara Micro projects hosted on Platform.sh.
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+Payara Micro is an open source, lightweight middleware of choice for containerized Java EE (Jakarta EE) microservices deployments.
```
Comment (Crell) [inline templates/microprofile-payara/files/README.md]: Open Source is capitalized.

"middleware of choice" sounds like an endorsement, which we should be avoiding.  Do you mean "middleware-based framework"?

File: templates/microprofile-tomee/files/README.md [hunk 6]
```diff
@@ -1,14 +1,21 @@
-# Apache TomEE MicroProfile template for Platform.sh
+# Apache TomEE MicroProfile for Platform.sh
 
-This project provides a starter kit for Apache TomEE MicroProfile projects hosted on [Platform.sh](http://platform.sh).
+This project provides a starter kit for Apache TomEE Eclipse MicroProfile projects hosted on Platform.sh.
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+Apache TomEE is the Eclipse MicroProfile flavor that uses several Apache Project flavors such as Apache Tomcat, Apache OpenWebBeans and so on.
```
Comment (Crell) [inline templates/microprofile-tomee/files/README.md]: "several Apache projects such as..."  Those aren't flavors, they're distinct projects.

File: templates/spring-boot-maven-mysql/files/README.md [hunk 7]
```diff
@@ -1,14 +1,22 @@
...
+This project provides a starter kit for Spring Boot Maven with MySQL projects hosted on Platform.sh.
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+The Spring Framework is an application framework and inversion of control container for the Java platform. The framework's core features can be used by any Java application, but there are extensions for building web applications on top of the Jakarta EE platform. Spring Boot makes it easy to create stand-alone, production-grade Spring based Applications that you can "just run".
 
-## Platform.sh configuration files
+## Services
+
+* Java 8
+* mysql 10.2
```
Comment (Crell) [inline templates/spring-boot-maven-mysql/files/README.md]: MariaDB

File: templates/spring-boot-maven-mysql/files/.platform.template.yaml [hunk 8]
```diff
@@ -23,7 +23,7 @@ info:
   # The human-readable name of the template.
   name: Spring Boot, Maven, Mysql
```
Comment (Crell) [inline templates/spring-boot-maven-mysql/files/.platform.template.yaml]: It occurs to me that we should start differentiating MySQL and MariaDB more now that we offer both.  Which means... this template should either switch to use Oracle-MySQL, or it should be renamed and the labeling specify MariaDB.

I'm good with either.  Possibly off topic for this PR, though.

File: templates/spring-boot-maven-mysql/files/README.md (hunk 7 above)
Comment (Crell) [reply templates/spring-boot-maven-mysql/files/README.md]: Or at least properly capitalized (see previous).

File: templates/spring-boot-maven-mysql/files/README.md [hunk 9]
```diff
@@ -1,14 +1,22 @@
-# Spring Boot Maven template for Platform.sh
+# Spring Boot Maven for Platform.sh
 
-This project provides a starter kit for Spring Boot Maven with MySQL projects hosted on [Platform.sh](http://platform.sh).
+This project provides a starter kit for Spring Boot Maven with MySQL projects hosted on Platform.sh.
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+The Spring Framework is an application framework and inversion of control container for the Java platform. The framework's core features can be used by any Java application, but there are extensions for building web applications on top of the Jakarta EE platform. Spring Boot makes it easy to create stand-alone, production-grade Spring based Applications that you can "just run".
```
Comment (Crell) [inline templates/spring-boot-maven-mysql/files/README.md]: The last sentence is extraneous.

File: templates/spring-mvc-maven-mongodb/files/README.md [hunk 10]
```diff
@@ -1,18 +1,26 @@
-# Spring MVC Maven template for Platform.sh
+# Spring MVC Maven for Platform.sh
 
-This project provides a starter kit for Spring MVC Maven with MongoDB projects hosted on [Platform.sh](http://platform.sh).
+This project provides a starter kit for Spring MVC Maven with MongoDB projects hosted on Platform.sh.
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+The Spring Framework is an application framework and inversion of control container for the Java platform. The framework's core features can be used by any Java application, but there are extensions for building web applications on top of the Jakarta EE platform. Spring Boot makes it easy to create stand-alone, production-grade Spring based Applications that you can "just run".
```
Comment (Crell) [inline templates/spring-mvc-maven-mongodb/files/README.md]: The last sentence is extraneous.

Comment (otaviojava) [reply templates/spring-mvc-maven-mongodb/files/README.md]: Thank you, done.
What do you think now?
I reduced the description also.

File: templates/spring-boot-maven-mysql/files/README.md (hunk 7 above)
Comment (otaviojava) [reply templates/spring-boot-maven-mysql/files/README.md]: Done, thank you

**Pull request merged**